<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>495</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test_TCP</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="IP_label">
                <property name="text">
                 <string>IPv4:</string>
                </property>
                <property name="buddy">
                 <cstring>IPv4_line_edit</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="IPv4_line_edit">
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="whatsThis">
                 <string/>
                </property>
                <property name="text">
                 <string>192.168.1.1</string>
                </property>
                <property name="maxLength">
                 <number>20</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="slave_label">
                <property name="text">
                 <string>Slave:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="slave_line_edit">
                <property name="maximumSize">
                 <size>
                  <width>50</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="text">
                 <string>1</string>
                </property>
                <property name="maxLength">
                 <number>10</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="port_label">
                <property name="text">
                 <string>Port:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="port_line_edit">
                <property name="text">
                 <string>502</string>
                </property>
                <property name="maxLength">
                 <number>8</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="rd_reg_label">
                <property name="text">
                 <string>Reg:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="rd_reg_line_edit">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="rd_count_label">
                <property name="text">
                 <string>Count</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="rd_count_line_edit">
                <property name="text">
                 <string>1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rd_button">
                <property name="text">
                 <string>Read</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="wr_reg_label">
                <property name="text">
                 <string>Reg:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="wr_reg_line_edit">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="wr_val_label">
                <property name="text">
                 <string>Value</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="wr_val_line_edit">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="wr_button">
                <property name="text">
                 <string>Write</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="connect_button">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="log_plain_text_edit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
